animation-worklet/idlharness.any.js

```diff
@@ -6,7 +6,7 @@
 // https://wicg.github.io/animation-worklet/
 
 idl_test(
-  ['animation-worklet'],
+  ['css-animation-worklet'],
```

Comment:
So the current url (and directory) is css-animationworklet, with no second '-'. I guess the question is whether AnimationWorklet is considered one word or two. Looks like it is two words in all the prose, so I would consider the Houdini directory (and the url) to be missing a hyphen. 

Cc @majido

Reply:
Agree that does look a bit funny.

Reply:
Animation Worklet is considered two words. So "css-animation-worklet" is correct. The old `css-animationworklet` directory should really be renamed to `css-animation-worklet`. I will start a houdini discussion to see how we can do it with least breakage. 

So this change is fine. I will make sure we update WPT if we change the directory and URL.

Reply:
I have filed: https://github.com/w3c/css-houdini-drafts/issues/959